_posts/2025-02-01-next-scala-lts.md

```diff
@@ -0,0 +1,130 @@
+---
+category: announcement
+permalink: /news/next-scala-lts.html
+title: "Next Scala 3 LTS version"
+by: Tomasz Godzik
+---
+
+**TLDR;**
+
+An API used in the `lazy val` implementation will be removed in a future JDK.
+Its replacement is only available on JDK 9+. Therefore, the Scala 3 Next minors
+in 2025 Q4 and the next LTS will drop JDK 8 support.
```

Comment:
```suggestion
in 2025 Q4 and the next LTS will drop JDK 8 support (and perhaps 11 as well).
```

Reply:
btw I still think the TL;DR is too technical and it should just read:

> An API used in the `lazy val` implementation will be removed in a future JDK. Its replacement is only available on JDK 9+. Therefore, the Scala 3 Next minors in 2025 Q4 and the next LTS will drop JDK 8 support.

at which point it might as well just be merged with the "Intro" section rather than being separate.

I do not think the `lazy val` detail is worth foregrounding so strongly. If it was mainly about that, we wouldn't be considering dropping 11 too.

Reply:
What happened to the consideration of going directly to `21`?

Reply:
> btw I still think the TL;DR is too technical and it should just read:

Yeah, that is probably better, changed.